.circleci/config.yml

```diff
@@ -101,7 +103,9 @@ jobs:
       - run:
           name: Deploy Snapshots to S3
           command: |
-            for f in ~/workspace/snapshot/*.zip; do
+            cd ~/workspace/snapshots
+            du -ah --apparent-size *
```

Comment:
Why do we need it?

Reply:
No one, I guess. It's just for making 1.6 look like 2.0.